### PR TITLE
Fix Phantom branch reference in Join proxy

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
NO

#### Description
Fix logo links to use develop branch in case we ever clean up branches. Also fix Radarr reference, as much as I want to leave it ;)

### CODE REVIEW

**Comment by bakerboy448** [comment]
technically just fixing you're own mistake 😛

**Comment by Qstick** [comment]
> technically just fixing your own mistake 😛

No mistakes were made… I didn’t pull it

File: src/NzbDrone.Core/Notifications/Join/JoinProxy.cs [hunk 1]
```diff
@@ -94,8 +94,8 @@ private void SendNotification(string title, string message, HttpMethod method, J
             var request = requestBuilder.AddQueryParam("apikey", settings.ApiKey)
                           .AddQueryParam("title", title)
                           .AddQueryParam("text", message)
-                          .AddQueryParam("icon", "https://cdn.rawgit.com/Sonarr/Sonarr/phantom-develop/Logo/256.png") // Use the Radarr logo.
-                          .AddQueryParam("smallicon", "https://cdn.rawgit.com/Sonarr/Sonarr/phantom-develop/Logo/96-Outline-White.png") // 96x96px with outline at 88x88px on a transparent background.
+                          .AddQueryParam("icon", "https://cdn.rawgit.com/Sonarr/Sonarr/develop/Logo/256.png") // Use the Sonarr logo.
```
Comment (Taloth) [inline src/NzbDrone.Core/Notifications/Join/JoinProxy.cs:97]: maybe we should update this to `/main/` instead of `/develop/`

Comment (Qstick) [reply src/NzbDrone.Core/Notifications/Join/JoinProxy.cs:97]: Main may make sense, as in the case of a logo change it'll be right for most of the user base all the time.

**Comment by Qstick** [comment]
@Taloth fixed

**Pull request merged**